app/src/main/AndroidManifest.xml

```diff
@@ -10,15 +13,19 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name=".ChatActivity"
+            android:name=".SplashActivity"
```

Comment:
Well it redirects the user. Its use will be visible only in further implementations.